docs/conf/portland/2022/cfp.rst

```diff
@@ -0,0 +1,185 @@
+:template: {{year}}/generic.html
+
+Call for Proposals
+==================
+
+Hello hello, fellow documentarians! It's that time of year again: We’re very excited to announce that we are now accepting talk proposals for our next {{ city }} conference, coming up on {{date.main}}. We're running a `hybrid conference <https://www.writethedocs.org/conf/portland/2022/news/welcome/>`__ this year, so you can speak or attend from anywhere, but folks will also be there in person in {{ city }}. We will use regular {{ city }} conference times for the scheduling.
+
+    For speakers this means you'll be writing and recording your talk *before the event*, the same as for our virtual conference. We'll have a videographer around to help with that if you want it. we're working on the exact details of a moderated remote live Q&A after your talk.
```

Comment:
```suggestion
For speakers, this means you'll be writing and recording your talk *before the event*, the same as for our virtual conference. Our video producer will be available to help with recording if you want it. We're working on the exact details of a moderated remote live Q&A after your talk.
```

Reply:
I removed the spaces before this paragraph but then I looked at the preview and saw that it ended up being a different font and alignment, so feel free to edit my suggestion and add the spaces back if you want :)